packages/apfelxx/variables.sh

```diff
@@ -1,5 +1,4 @@
 BRANCH="resummation-scales"
-VERSION="7572c1b"
 URL="https://github.com/vbertone/apfelxx/tarball/$BRANCH"
```

Comment:
In principle, I believe that it should be enough to write here:
```suggestion
URL="https://github.com/vbertone/apfelxx/tarball/$VERSION"
```
and it should work out of the box :)

Reply:
https://github.com/vbertone/apfelxx/tarball/7572c1b

Reply:
Yes, this is indeed also an option. But given that we'd want to always point to the tip of the branch I'd say we want to be commit-independent (at least until Valerio merges that branch).

Reply:
Personally, I'd prefer to avoid: you will install anyhow a limited number of times, but in this way it will be impossible to know which version you were using after something has gone wrong (unless you modify the code adding the hash of the commit, and propagate to a log every time).

Reply:
Ok, I think this would also be fine. We'll just need to make sure then that every time major changes happen in that branch we'd reflect them here.